src/libsync/platform_win.cpp

```diff
@@ -29,11 +31,20 @@ WinPlatform::~WinPlatform()
 
 void WinPlatform::setApplication(QCoreApplication *application)
 {
-    Q_UNUSED(application)
-
     // Ensure OpenSSL config file is only loaded from app directory
     const QString opensslConf = QCoreApplication::applicationDirPath() + QStringLiteral("/openssl.cnf");
     qputenv("OPENSSL_CONF", opensslConf.toLocal8Bit());
+
+    // The Windows style still has pixelated elements with Qt 5.6,
+    // it's recommended to use the Fusion style in this case, even
+    // though it looks slightly less native. Check here after the
+    // QApplication was constructed, but before any QWidget is
+    // constructed.
+    if (auto guiApp = qobject_cast<QGuiApplication *>(application)) {
+        if (guiApp->devicePixelRatio() > 1) {
+            QApplication::setStyle(QStringLiteral("fusion"));
+        }
+    }
```

Comment:
Not a fan of this. We should probably just set fusion all the time. But on the other hand, I don't really care.